source/catalog.html

```diff
@@ -37,11 +40,11 @@
 
     <main class="page__main page-main">
       <div class="page-main__wrapper">
-        <h1 class="main-title">Каталог продукции</h1>
+        <h1 class="page-main__title main-title page-main__title--catalog">Каталог продукции</h1>
 
         <ul class="product-catalog">
           <li class="product-card">
-            <a href="#">
+            <a class="product-card__product-link" href="#">
```

Comment:
А зачем 2 раза писать `product`? У нас уже обозначено в названии блока что это карточка продукта, соответственно и в класс элемента это попадает:
```suggestion
            <a class="product-card__link" href="#">
```